app/src/main/java/io/apicurio/registry/events/http/HttpEventSinkConfiguration.java

```diff
@@ -34,6 +35,7 @@
 public class HttpEventSinkConfiguration {
 
     @ConfigProperty(name = "registry.events.ksink")
+    @Info( category = "events", description = "Events ksink enabled", availableSince = "2.0.0.Final")
     Optional<String> ksink;
```

Comment:
Should this be `Events Kafka sink enabled` or is a `ksink` something else?

Reply:
👍